src/components/Menu/tests/Menu.test.tsx

```diff
@@ -31,6 +31,12 @@ describe('<Menu />', () => {
       expect(wrapper.find(MenuAction)).toHaveLength(mockActions.length);
     });
 
+    it('does not render <MenuAction /> when there are no actions', () => {
+      const wrapper = mountWithAppProvider(<Menu {...mockProps} />);
+
+      expect(wrapper.find(MenuAction)).toHaveLength(0);
+    });
+
```

Comment:
This and the change below was to handle these partially covered paths

|:computer:|
|-|
|<img width="1343" alt="Screen Shot 2019-06-12 at 10 22 59 AM" src="https://user-images.githubusercontent.com/1403188/59360237-bdc61280-8cfd-11e9-885a-1abd0538d3c3.png">|